packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -42,8 +42,18 @@ const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
     );
 };
 
+const Loader = () => (
```

Comment:
@thisyahlen-deriv we can move it to `packages/tradershub/src/components/SkeletonLoader/CurrencySwitcherLoader`

Reply:
I dont see the need to put this into a new file since its being used just here. Plus we are not using third party libraries for the loader. IMO its more efficient to put it here

Reply:
I don't completely agree with you @thisyahlen-deriv I think with a few tweaks you can make generalize and import it to the files that are using it.

Reply:
For me both ways works it's a matter of preference. 🤷🏼‍♂️

Reply:
@thisyahlen-deriv Having a loader directory to put all the loaders in one place looks cleaner to me.